CODE_REVIEW.md

```diff
@@ -0,0 +1,31 @@
+## Code Review ##
+
+### Issues ###
+
+* BookSearchComponent: When you search for a new book and the search result panel opens, the previous search results are not cleared and are visible.
```

Comment:
This is more of a UX issue than an issue with the code. Why does it do this? What would you do instead?